### ISSUE TITLE
Opening the filter popover menu

### ISSUE THREAD

**Issue by colton-dewberry** [microsoft/vscode-data-wrangler#216]
Type: <b>Bug</b>

# Behaviour

## Expected vs. Actual

Expected: I am able to select inputs in the filter pop-over menu
Actual: Attempting to select an input crashes and closes the pop-over

## Steps to reproduce:

1. Select the filter option
2. Move cursor towards the bottom of the pop-over

# Diagnostic data

-   Jupyter extension version: 2024.5.0
-   Python extension version: 2024.8.1
-   .NET Install Tool for Extension Authors extension version: 2.0.7
-   Python package dependencies:

```json
{
  "installed": {
    "pandas": "1.3.5"
  },
  "required": {
    "pandas": "0.25.2"
  },
  "unsatisfied": []
}
```

-   Entrypoint: JupyterDataViewer
-   Active mode: dataViewer

Extension version: 1.4.1
VS Code version: Code 1.90.2 (Universal) (5437499feb04f7a586f677b155b039bc2b3669eb, 2024-06-18T22:37:41.291Z)
OS version: Darwin x64 19.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) W-2170B CPU @ 2.50GHz (28 x 2500)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|7, 5, 5|
|Memory (System)|32.00GB (0.38GB free)|
|Process Argv|--crash-reporter-id bf30415b-cd72-4498-b8cd-f9d5c7640076|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
vscrp:30673768
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonnoceb:30805159
asynctok:30898717
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
jchc7451:31067544
chatpanelt:31048053
dsvsc021:30996838
f3je6385:31013174
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
dwnewjupyter:31046869
newcmakeconfigv2:31071590
legacy_priority:31082724

```

</details>

**Comment by pwang347** [comment]
Hi @colton-dewberry, thanks for reporting this issue! Could you please share any errors you see in the console?
Ie. from the following menu:
<img width="381" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/2cc22f4f-2c8a-4848-9f3f-1e80758dc885">

And then in the console tab:
<img width="382" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/9d3f9b71-bcd9-4d0f-aa46-2106a3e113fc">

A GIF would be very appreciated as well, but not as important. Thanks!

**Comment by pwang347** [comment]
Hi @colton-dewberry! We've identified the issue and have released a fix in both stable (v1.4.2+) and pre-release (v1.5.3+) channels for it. Please give the latest version a try and let me know if the issue is still occurring for you.

Thanks!
